app/assets/javascripts/discourse/app/templates/user-invited-show.hbs

```diff
@@ -99,6 +115,19 @@
                         {{d-button icon="sync" action=(action "reinvite") actionParam=invite label="user.invited.reinvite"}}
                       {{/if}}
                     </td>
+                  {{else if inviteLinks}}
+                    <td>{{d-button icon="link" action=(route-action "editInvite" invite.invite_link) label="user.invited.copy_link"}}</td>
+                    <td>{{format-date invite.created_at}}</td>
+                    <td>{{number invite.redemption_count}} / {{number invite.max_redemption_allowed}}</td>
+                    <td colspan="2">{{ invite.group_names }}</td>
+                    <td>{{raw-date invite.invite_link_expires_at leaveAgo="true"}}</td>
```

Comment:
does true need to be a string here ?

Reply:
Yes, seems like we need string here because that's what we are checking for.

https://github.com/discourse/discourse/blob/d615de9139aaede37dbb5f97d89e08b7541869ec/app/assets/javascripts/discourse/app/helpers/format-date.js#L15